### ISSUE TITLE
Displaying the name of the place in the car mode \ Отображение названий нп в режиме автомобиль

### ISSUE THREAD

**Issue by ikid0** [osmandapp/OsmAnd#11061]
# 🚀 feature request

Иногда, когда едешь через нп, центр (точка place) которого отдалён от трассы, на экране не видно через какой нп проезжаешь. Единственный вариант узнать название нп - масштабировать карту, что неудобно.

Я предлагаю как вариант в режиме автомобиль отображать тег [traffic_sign=city_limit](https://wiki.openstreetmap.org/wiki/Tag:traffic_sign=city_limit) как знак въезда в нп/выезда из нп. Можно стилизовать под знак из ПДД, брать name и если указано - цвет фона. Возможно также голосовое оповещение при проезде через эту точку.

=======
Google translate, sorry.

Sometimes, when you go through an place, the center (point of place) of which is far from the road, you cannot see on the screen which place you are passing through. The only way to find out the name of the place is to scale the map, which is inconvenient.

I suggest as an option in car mode  display tag [traffic_sign=city_limit](https://wiki.openstreetmap.org/wiki/Tag:traffic_sign=city_limit) as a sign of entry to / exit from place. You can style it as a sign from the PDD, take name and, if specified, the background color. Voice notification is also possible when driving through this point.
![63191 pv6uvc 1280](https://user-images.githubusercontent.com/3027212/109842909-fb9e6380-7c63-11eb-86e5-8d5c71d4d37c.jpg)

**Comment by Ungoose** [comment]
I agree, it would be a neat feature - getting a sign displayed in the same way OsmAnd currenly warns abour railway crossings.
